### PR TITLE
fix localization issues with default locations

### PR DESCRIPTION
**Pull request by marlokessler**
## What does this PR do?

This PR adds the localization function to location labels in BookingPage, AvailableEventLocations and EventSetupTab. It also replaces the labels for the Attendee Address location type with localization keys and add the keys to the English and German localization table. Finally, it corrects two other missspellings in the German localization table.

Fixes # (issue)
- localization issues with default locations.

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- [x] Test by trying it out

## Checklist
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: apps/web/components/booking/AvailableEventLocations.tsx [hunk 1]
```diff
@@ -23,8 +26,8 @@ export function AvailableEventLocations({ locations }: { locations: Props["event
               )}
               alt={`${eventLocationType.label} icon`}
             />
-            <Tooltip content={locationKeyToString(location)}>
-              <p className="truncate">{locationKeyToString(location)}</p>
+            <Tooltip content={t(locationKeyToString(location) ?? "")}>
+              <p className="truncate">{t(locationKeyToString(location) ?? "")}</p>
```
Comment (PeerRich) [inline apps/web/components/booking/AvailableEventLocations.tsx:30]: what does ?? "" do? havent seen that before with t() function

Comment (marlokessler) [reply apps/web/components/booking/AvailableEventLocations.tsx:30]: It serves as a fallback, if `locationKeyToString` returns`undefined`.

**Review by zomars — APPROVED** [review]
Thank you for your contribution 🙏

**Comment by zomars** [comment]
Seems like we have some conflicts to solve first.

**Comment by marlokessler** [comment]
> Seems like we have some conflicts to solve first.

Done👍🏼

**Comment by PeerRich** [comment]
why are we merging into [calcom:pr/5569](https://github.com/calcom/cal.com/tree/pr/5569) and not into main?

**Comment by marlokessler** [comment]
> why are we merging into [calcom:pr/5569](https://github.com/calcom/cal.com/tree/pr/5569) and not into main?

Ooops, sry. changed the merge branch.

**Comment by PeerRich** [comment]
@marlokessler nice thank you, lastly can you update your base branch and remove branch protection so we can update the branch too?


CleanShot 2022-12-26 at 11 28 04

**Comment by marlokessler** [comment]
> @marlokessler nice thank you, lastly can you update your base branch and remove branch protection so we can update the branch too?
> 
> CleanShot 2022-12-26 at 11 28 04

I updated the base branch👍🏼 The repo is not covered with any branch protection rules, but I added you to the repo, so that you have write access too

**Comment by marlokessler** [comment]
> This PR currently has a merge conflict. Please resolve this and then re-add the `automerge` label.

Done

**Pull request merged**